addons/website_profile/views/website_profile.xml

```diff
@@ -315,8 +315,11 @@
                 <t t-else="">
                     <t t-set="user_points" t-value="0"/>
                 </t>
-                <path class="o_pc_circle_bg" d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" />
-                <path class="o_pc_circle" t-attf-stroke-dasharray="#{user_points}, 100" d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" stroke="url(#gradient)" mask="url(#mask)"/>
+                <g data-bs-toggle="tooltip"
+                    t-attf-title="{{ ('{0}/{1} xp'.format(user.karma, next_rank_id.karma_min)) if next_rank_id else ''}}">
```

Comment:
Maybe using f-strings would be more readable here.

Reply:
I tried but did not work. Could you provide an example?

Reply:
Maybe it's not supported in XML.

In that case how about a regular:
`{{ '%s/%s xp' % (user.karma, next_rank_id.karma_min) if next_rank_id else ''}}`

?

Reply:
Done :)